gravitee-apim-console-webui/src/management/settings/custom-user-fields/migrated/custom-user-fields-migrated.component.ts

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, DestroyRef, inject, OnInit } from '@angular/core';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { MatDialog } from '@angular/material/dialog';
+import { GIO_DIALOG_WIDTH, GioConfirmDialogComponent, GioConfirmDialogData } from '@gravitee/ui-particles-angular';
+import { BehaviorSubject, switchMap } from 'rxjs';
+import { debounceTime, distinctUntilChanged, filter, tap } from 'rxjs/operators';
+import { isEqual, orderBy, get } from 'lodash';
+
+import {
+  CustomUserFieldsDialogComponent,
+  CustomUserFieldsDialogData,
+  CustomUserFieldsDialogResult,
+} from './dialog/custom-user-fields-dialog.component';
+
+import { GioTableWrapperFilters, Sort } from '../../../../shared/components/gio-table-wrapper/gio-table-wrapper.component';
+import { CustomUserFieldsService } from '../../../../services-ngx/custom-user-fields.service';
+import { SnackBarService } from '../../../../services-ngx/snack-bar.service';
+import { CustomUserField } from '../../../../entities/customUserFields';
+
+@Component({
+  selector: 'app-custom-user-fields-migrated',
+  templateUrl: './custom-user-fields-migrated.component.html',
+  styleUrl: './custom-user-fields-migrated.component.scss',
+})
+export class CustomUserFieldsMigratedComponent implements OnInit {
+  private destroyRef: DestroyRef = inject(DestroyRef);
+  public isListLoading = true;
+  public isLoading = false;
+  public customUserFields: CustomUserField[] = [];
+  public customUserFieldsFiltered: CustomUserField[] = [];
+  public displayedColumns: string[] = ['key', 'label', 'values', 'actions'];
+  public nbTotalInstances = this.customUserFields.length;
+  public filters: GioTableWrapperFilters = {
+    pagination: { index: 1, size: 10 },
+    searchTerm: '',
+    sort: { active: null, direction: null },
+  };
+
+  private filters$ = new BehaviorSubject<GioTableWrapperFilters>(this.filters);
```

Comment:
unused as you have Console pagination only. Imho use-full when you have res-api pagination & filters. 

WDIT to a simple void refresh$ BehaviorSubject ?

Reply:
removed subject, refresh$ not needed IMHO